iml-manager-cli/Cargo.toml

```diff
@@ -21,8 +21,8 @@ indicatif = "0.14"
 lazy_static = "1.4"
 number-formatter = {path = "../number-formatter", version = "0.1"}
 prettytable-rs = "0.8"
-reqwest = {version = "0.10", features = ["default-tls", "json"]}
-serde = {version = "1", features = ["derive"]}
+reqwest = { version = "0.10", features = ["rustls-tls", "json"] }
+serde = { version = "1", features = ["derive"] }
```

Comment:
Not sure where I'm getting it from, but my vscode has an autoformatter for .toml that removes the space between the braces.

Reply:
Done